charts/port-ocean/values.yaml

```
@@ -91,4 +91,7 @@ integration:
 
 selfSignedCertificate:
   enabled: false
-  certificate: ""
\ No newline at end of file
+  certificate: ""
+  secret: ""
+  useExistingSecret: false
+  key: crt
```

Just a minor issue over here

```suggestion
  secret: ""
    # name: ""
    # key: crt
```